### ISSUE TITLE
run `inv manage migrate_cloudinary` against staging

### ISSUE THREAD

**Issue by Pomax** [mozilla/foundation.mozilla.org#6284]
Follow-up to #6248 - pre-prod-run test to make sure it actually did what we printed on the tin.

**Comment by Pomax** [comment]
@KalobTaulien do we want to update this command so that it creates the images in a get-or-create PNI collection?

**Comment by KalobTaulien** [comment]
@Pomax Yes I'll add that into this management command. Putting images into Collections is a good idea!

**Comment by Pomax** [comment]
staging with PNI image collection works, so we just need to run this against prod now.

**Comment by Pomax** [comment]
done running in production, too. turning off USE_CLOUDINARY in production.

**Comment by Pomax** [comment]
This blew up - I've set both staging and prod to `USE_CLOUDINARY = True` again, and we'll have to do something about image rendition locks

**Comment by jamilasnell** [comment]
Moving up the estimate from 0 - 16 (this includes the amount of work that has already been done on it)

**Comment by Pomax** [comment]
Updated result: this runs quite fine on staging, but loading the PNI homepage then causes page serving to time out due to watail trying to create image renditions when the PNI homepage gets requested.

**Comment by Pomax** [comment]
See https://github.com/mozilla/foundation.mozilla.org/issues/6373

**Comment by Pomax** [comment]
blocked by the PNI homepage taking too long to load due to the number of database queries required to service 126 x 8 thumbnail sizes.
